docs/website/docs/deployment-configurations/gpu-vulkan.md

```diff
@@ -6,12 +6,22 @@ operating systems, including Android, Linux, and Windows. Vulkan is also
 cross-vendor: it is supported by most GPU vendors, including AMD, ARM, Intel,
 NVIDIA, and Qualcomm.
 
-!!! todo
+<!-- TODO(??): when to use CPU vs GPU -->
 
-    Add IREE's GPU support matrix: what GPUs are supported; what GPUs are well
-    optimized; etc.
+## Support matrix
 
-<!-- TODO(??): when to use CPU vs GPU -->
+In IREE, right now we are still developing generic compiler transformations to
+benefit generating performant code for all hardware architectures. We are not
+at the stage of performing a lot of target specific optimizations. So in
+general, we expect all GPUs to see reasonable performance, but might not
+in the best shape.
```

Comment:
Wordsmithing a bit. WDYT?

```suggestion
As IREE and the compiler ecosystem it operates within matures, more
target specific optimizations will be implemented. At this stage, expect
reasonable performance across all GPUs and for improvements to be
made over time for specific vendors and architectures.
```

sections I'd particularly like to reword:

* "but might not in the best shape"
* Remove the strong focus on "we" (as project maintainers) vs "IREE" (the product users are reading about). Documentation for project developers can use "we" more often, but the user facing documentation should be more deliberate. For example, the website uses "we use github projects" in the roadmap section since that is talking about the team, and some of the building from source pages have "we expect it to work" for similar reasons.

Reply:
https://developers.google.com/style/pronouns#personal-pronouns